### DISCUSSION TITLE
Connectors to update/remove from the top asks

### DISCUSSION THREAD

**Discussion by paulculmsee** [microsoft/PowerPlatformConnectors#1601] [Ideas]
@natalie-pienkowska I saw ElectricityMap there, and reached out to them as per their TOS ages ago but they were not interested. Maybe we need a status field for each of the asks?

"After internal discussions, we sadly decided that we will have to push our collaboration further down our roadmap. If it is okay with you, we would connect again at a later point in time when we have more free capabilities and can discuss further details with seven sigma!"

**Comment by natalie-pienkowska** [comment]
Thanks for letting me know about this! Yes, either that (and we link to this discussion) or we remove it from the list of top asks. What do you think?
